HowTo install Linux (Ubuntu) tools for C++ programming with Eclipse IDE/readme.md

```diff
@@ -66,14 +66,44 @@ Categories=Development;IDE
 Name[en]=eclipse.desktop
 ```
 
+The shortcut must be executable. You can set this with the following command:
+
+```sh
+sudo  chmod 744 ~Desktop/eclipse.desktop
+```
+
 Create a symlink to make eclipse avaible from the console:
 
 ```sh
 cd /usr/local/bin
 sudo ln -s /opt/eclipse/eclipse
 ```
 
-**Eclipse** is now ready to use.
+Use the following command to assign the Eclipse icon to the shortcut:
+
+```sh
+sudo cp /opt/eclipse/icon.xpm /usr/share/pixmaps/eclipse.xpm
+```
+Java is required to start Eclipse. It can be installed as a package
+
+```sh
+sudo apt-get install openjdk-8-jdk
+```
+
```

Comment:
Java installation is described earlier in the readme. Thus, this part can be removed.